pkgs/development/haskell-modules/configuration-ghc-7.10.x.nix

```diff
@@ -182,6 +186,9 @@ self: super: {
   # https://github.com/well-typed/hackage-security/issues/158
   hackage-security = dontHaddock (dontCheck super.hackage-security);
 
+  # Fixed in 1.0.5, remove if stackage and configuration-hackage2nix.yaml catched up
```

Comment:
That happens within a week or less. Is it okay to avoid adding this override temporarily for such a short period of time?